src/JLD2.jl

```diff
@@ -56,6 +56,14 @@ include("bufferedio.jl")
 include("misc.jl")
 include("superblock.jl")
 
+is_win7() = Sys.iswindows() && Sys.windows_version().major <= 6 && Sys.windows_version().minor <= 1
+# Windows 7 doesn't support mmap, falls back to IOStream
+const DEFAULT_IOTYPE = if is_win7()
+    IOStream
+else
+    JLD2.MmapIO
```

Comment:
I think it should be

```suggestion
    MmapIO
```